build_tools/python/benchmark_suites/iree/armv8_a_benchmarks.py

```diff
@@ -38,25 +38,23 @@ class Android_ARMv8_A_Benchmarks(object):
         tags=["default-flags"],
         compile_targets=[ARMV8_A_CPU_TARGET],
     )
-    MMT4D_COMPILE_CONFIG = iree_definitions.CompileConfig.build(
+    DATA_TILING_COMPILE_CONFIG = iree_definitions.CompileConfig.build(
         id=unique_ids.IREE_COMPILE_CONFIG_ANDROID_ARMV8_2_A_GENERIC_MMT4D,
-        tags=["experimental-flags", "mmt4d"],
+        tags=["experimental-flags", "data-tiling", "ukernel"],
         compile_targets=[ARMV8_A_CPU_TARGET],
         extra_flags=[
             "--iree-opt-data-tiling",
-            "--iree-flow-enable-fuse-padding-into-linalg-consumer-ops",
-            "--iree-llvmcpu-enable-pad-consumer-fusion",
```

Comment:
Why drop this? (Two questions - why drop this at all, and why do this as part of the same PR?)

Reply:
I chatted with @hanhanW about what experimental flags to use and his guidance was we want `--iree-opt-data-tiling` and `--iree-llvmcpu-enable-microkernels` and the padding config flags can be dropped since it only applies to convolution models.

Reply:
I'm happy to change it to what makes most sense, but it would be good to enable the data+tiling and microkernel path in the mobile benchmarks (currently it's only enabled on x86).

Reply:
Thanks for the explanation!